experiment/simulation/index.html

```diff
@@ -1,21 +1,235 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<!-- Add CSS at the head of HTML file -->
-    <link rel="stylesheet" href="./css/main.css">
-</head>
-<body>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.8-b/css/style.min.css"
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
 
-		<!-- Your code goes here-->
+  <body class="page has-text-centered">
+    <div class="columns is-centered is-flex">
+      <div class="column has-text-centered">
+        <div class="v-collapsible-instruction-container">
+          <div class="v-collapsible">Instructions</div>
+          <div id="procedure-message "class="v-content">
+            <p style="font-size: 0.9rem">
+              <ol>
+                <li>Use the Reset button to reset the simulation</li>
+                <li>Click on Next button to go to next iteration</li>
+                <li>Click on Previous button to go to previous iteration</li>
+                <li>
+                  Change the positions of rectangle coordinates and line
+                  coordinates
+                </li>   </ul>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <br />
 
-	<p><li>This simulation has some <a href="../../../Introduction.html" target="PARENT">software dependencies</a>. We highly recommend you to use our customized <a href="https://vlead.vlabs.ac.in/development/#virtual-box" target="blank">VirtualBox</a> for the smooth running of these simulations.</li></p>
+    <div
+      class="columns is-centered is-variable is-1-mobile is-flex is-flex-wrap-wrap workspace left"
+    >
+      <div
+        id="left"
+        class="column has-text-centered is-2-desktop is-5-tablet is-5-mobile apparatus"
+      >
+        <div class="v-datalist-container">
+          <div class="v-datalist-title is-size-8-mobile">Input</div>
+          <div class="v-datalist-row">
+            <div style="float: left" class="is-size-7-mobile v-content">
+              Line Co-ordinates:
+            </div>
+            <div>
+              <div class="v-datalist-row">
+                <p class="float:is-centered">Clipping Rectangle Coordinates</p>
+                <br />
+                <div style="display: is-flex">
```

Comment:
display: is-flex doesn't mean anything, it's either display: flex or class="is-flex". Use the class